scripts/check-public-api.ts

```diff
@@ -4,11 +4,12 @@ import { GlobSync } from 'glob';
 import { createLogger, flatten, unixEOL } from '@sap-cloud-sdk/util';
 import mock from 'mock-fs';
 import { CompilerOptions } from 'typescript';
+/* eslint-disable import/no-internal-modules */
```

Comment:
there is a eslint-disable-next-line - I assume this does not work here?

Reply:
eslint-disable-next-line doesn't work when the whole import statement is not in a single line.